### PR TITLE
SubCategoryDTO work and tested

### PR DESCRIPTION
**Pull request by neoed**

### CODE REVIEW

File: src/main/java/com/inqoo/trainingservice/app/subcategory/SubCategoryConverter.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.inqoo.trainingservice.app.subcategory;
+
+import org.springframework.stereotype.Component;
+
+@Component
```
Comment (mjasinskee) [inline src/main/java/com/inqoo/trainingservice/app/subcategory/SubCategoryConverter.java:5]: Usualy util classes like converters doesn't have to be Spring beans. Please consider construct it by yourself or change to static calls

File: src/test/java/com/inqoo/trainingservice/app/service/SubcategoryServiceIT.java [hunk 2]
```diff
@@ -17,93 +22,137 @@
...
+
 @SpringBootTest
 @Transactional
 class SubcategoryServiceIT {
     @Autowired
     private SubcategoryService subcategoryService;
     @Autowired
     private CategoryService categoryService;
+    @Autowired
+    private CategoryConverter categoryConverter;
```
Comment (mjasinskee) [inline src/test/java/com/inqoo/trainingservice/app/service/SubcategoryServiceIT.java:34]: Try to avoind using converters as Spring beans. Try to use them by yourself or change to static calls

**Review by mjasinskee — CHANGES_REQUESTED** [review]

File: src/main/java/com/inqoo/trainingservice/app/subcategory/SubCategoryConverter.java (hunk 1 above)
Comment (neoed) [reply src/main/java/com/inqoo/trainingservice/app/subcategory/SubCategoryConverter.java:5]: done

File: src/test/java/com/inqoo/trainingservice/app/service/SubcategoryServiceIT.java (hunk 2 above)
Comment (neoed) [reply src/test/java/com/inqoo/trainingservice/app/service/SubcategoryServiceIT.java:34]: done

**Review by mjasinskee — APPROVED** [review]

**Pull request merged**